### ISSUE TITLE
Issues with flex-flow: row wrap, items are cascaded

### ISSUE THREAD

**Issue by jonas-meyer** [schnerring/hugo-mod-json-resume#3]
Thanks for the module, very helpful to set up my personal website. I'm using the hugo-coder theme with this module and the following custom css: 

```css
.jr__item-meta {
  align-items: center;
  flex-flow: row wrap;
}

.jr__date,
.jr__date-range,
.jr-work__location {
  flex-grow: 1;
  text-align: right;
}
```

I'm getting the following output:
<img width="828" alt="Screenshot 2023-03-17 at 17 17 25" src="https://user-images.githubusercontent.com/19151471/225974153-2d3cc6ed-89df-4226-adbd-9f8e7cf046e4.png">

According to the README.md, the items in item-meta should align properly, is this something that you would know the solution to? Otherwise I'll need to read up on some css!

**Comment by schnerring** [comment]
My guess would be that the styles of the base theme interfere with the `jr__` elements?

I use the module successfully in my [Hugo Gruvbox](https://github.com/schnerring/hugo-theme-gruvbox) theme. Here's the bit where I added the custom CSS like described in the README:

https://github.com/schnerring/hugo-theme-gruvbox/blob/main/assets/css/critical/45-json-resume.css

You can find a demo here if you wanna look at in in the browser dev tools:

https://hugo-theme-gruvbox.schnerring.net/cv/

Is your repo public so I can run a quick `hugo server` to reproduce the issue locally? It's been a while that I've fiddled with CSS, however. so I'll probably have to read up on some CSS like you, hehe.

**Comment by jonas-meyer** [comment]
Hi Michael, 

Sorry for the late response, I've had a look in the browser dev tools and can't find anything that interferes with the jr__ elements. A public repo can be found here: https://github.com/jonas-meyer/personal-website-hugo

Let me know if you're able to reproduce the issue!

**Comment by schnerring** [comment]
Sorry for taking so long to get back to you! I saw on your website that the issue still persists. I tried reproducing the issue using the repo you provided. 

I ran the following commands to start up the site:

```
git submodule update --init --recursive
npm ci
hugo server
```

But `http://localhost:1313/resume/` looks like this 🤷

![image](https://github.com/schnerring/hugo-mod-json-resume/assets/3743342/fe84773d-bc69-4919-8152-9300363ebd9a)

Opposed to from `https://www.jonasmeyerohle.dev/resume/`:

![image](https://github.com/schnerring/hugo-mod-json-resume/assets/3743342/e2fa4491-34de-4070-991e-4369e676259d)

One thing I noticed, however, is that your site contains a file `assets/css/json-resume.css`. It just so happens that the file containing base CSS of `hugo-mod-json-resume` is named exactly the same. Try renaming your overrides to something like: `assets/css/json-resume-overrides.css`.

Let me know if that fixes the issue!
